utils/treesitter/grammar.js

```diff
@@ -402,7 +402,7 @@ module.exports = grammar({
       choice(
         $.generic_binding,
         seq($.ident, ':', $._expression),
```

Comment:
This line may now be deleted -- we don't allow `:` in the deduced parameter list at the moment, except after `self`.